core/components/heading.ts

```diff
@@ -153,7 +153,7 @@ function pluginsFactory({
 
           [keybindings.insertEmptyParaAbove]: insertEmptyParaAbove(),
           [keybindings.insertEmptyParaBelow]: insertEmptyParaBelow(),
-          [keybindings.toggleCollapse!]: toggleHeadingCollapse(),
+          // [keybindings.toggleCollapse]: toggleHeadingCollapse(),
```

Comment:
I commented this out because it make typing the statement very difficult, and since the key is null it would be removed from the keymap anyway.